contexts/authentification-token.js

```diff
@@ -28,8 +28,16 @@ export const AuthentificationContextProvider = props => {
   const checkUserRole = useCallback(async () => {
     try {
       const getUserRole = await authentificationRole(token)
-      setUserRole(getUserRole.role)
-      setIsAuthDataRecovering(false)
+
+      if (getUserRole.code === 403) {
+        localStorage.clear()
```

Comment:
Retire uniquement la clé du token.
```suggestion
        localStorage.removeItem(TOKEN_KEY)
```